### PR TITLE
decals_cats

### PR DESCRIPTION
**Pull request by Maquiner**

### CODE REVIEW

**Review by damonge — CHANGES_REQUESTED** [review]
Do we want to be consistent and save the signal maps too?

File: xcell/mappers/mapper_DECaLS.py [hunk 1]
```diff
@@ -31,25 +31,16 @@ def __init__(self, config):
         self.cat_data = []
```
Comment (damonge) [inline xcell/mappers/mapper_DECaLS.py:31]: I would make this `self.cat_data = None` (see comment below)

File: xcell/mappers/mapper_DECaLS.py [hunk 2]
```diff
@@ -58,21 +49,37 @@ def __init__(self, config):
...
 
-    def _get_angmask(self, cat):
-        bmask = hp.read_map(self.config['binary_mask'],
-                            verbose=False)
-        nside = hp.npix2nside(len(bmask))
-        ipix = hp.ang2pix(nside, cat['RA'], cat['DEC'],
-                          lonlat=True)
-        return bmask[ipix] > 0.
+    def get_catalogs(self):
+        if len(self.cat_data) == 0:
```
Comment (damonge) [inline xcell/mappers/mapper_DECaLS.py:53]: then make this
```
if self.cat_data == None:
    self.cat_data = []
```
Just to be consistent with the other mappers.

**Review by damonge — CHANGES_REQUESTED** [review]

File: xcell/mappers/mapper_DECaLS.py (hunk 2 above)
Comment (Maquiner) [reply xcell/mappers/mapper_DECaLS.py]: resolved!

**Review by damonge — APPROVED** [review]
LGTM!

**Pull request merged**